helm/prometheus-rules/templates/alerting-rules/job.rules.yml

```diff
@@ -22,6 +22,18 @@ spec:
         severity: notify
         team: {{ include "providerTeam" . }}
         topic: managementcluster
+    ## This alert triggers when the silence sync job did not schedule for more than 1 day.
+    - alert: SilenceJobHasNotBeenScheduledForTooLong
+      annotations:
+        description: '{{`CronJob {{ $labels.namespace }}/{{ $labels.cronjob }} has not been scheduled for more than 1 day.`}}'
+        opsrecipe: job-has-not-been-scheduled-for-too-long/
+      expr: (time() - kube_cronjob_status_last_schedule_time{cronjob="silence-operator-sync"} ) > 86400
```

Comment:
Does the `kube_cronjob_status_last_schedule_time` metrics tell us that the job actually succeeded ? I feel like it does not.

There is another metric `kube_job_status_succeeded` but it does not tell us when the job ran.

I would suggest we use a combination of both, so we know the job ran in the last day and there was at least 1 success.

Reply:
Maybe some ideas there https://lyz-code.github.io/blue-book/devops/kubernetes/kubernetes_jobs/#building-our-alert

Reply:
That's some interesting link

Reply:
Hey, look at the new metric that's coming! :wink: 
https://github.com/kubernetes/kube-state-metrics/pull/1732

Reply:
Interesting :)